src/components/OperationTag.js

```diff
@@ -78,18 +80,19 @@ export default class OperationTag extends React.Component {
               }
           </p>
 
-          <button
-            className="expand-operation"
-            title={showTag ? "Collapse operation": "Expand operation"}
-            onClick={() => layoutActions.show(isShownKey, !showTag)}
-            aria-expanded={showTag}
-            tabIndex={-1}
-          >
-            <svg className="arrow" width="20" height="20">
-              <use href={showTag ? "#large-arrow-down" : "#large-arrow"} xlinkHref={showTag ? "#large-arrow-down" : "#large-arrow"} />
-            </svg>
-          </button>
-        </h1>
+            <button
```

Comment:
I think we can make this a div, add a class for styling called `arrow-wrapper` - then add the styling:

```
.swagger-ui .opblock-tag .arrow-wrapper {
  padding: 0 30px;
}
```